doc/lua_api.txt

```diff
@@ -2634,6 +2633,19 @@ Some types may inherit styles from parent types.
 * tabheader
     * noclip - boolean, set to true to allow the element to exceed formspec bounds.
     * textcolor - color. Default white.
+* box
```

Comment:
Please keep the alphabetic sorting.

Reply:
The sorting isn't alphabetical as it is.  Does `s` come after `c` and before `d`?  I guess I'll fix all of them then.

Reply:
Oh. I only looked at the diff where I assumed it would be sorted. Thanks for sorting them all.